.github/workflows/ci.yml

```diff
@@ -1,21 +1,31 @@
 name: CI
-
 on:
-  push:
-    branches-ignore:
-      - develop
+  pull_request:
+    branches:
       - master
-      - 'release/**'
-
+      - develop
```

Comment:
The CI will not be triggered on other branches (if a pull request is made to a random branch, the danger action will not be called then). Is this the desired behavior ?

Reply:
No, you're right